code/modules/mob/living/simple_mob/subtypes/vore/bigdragon_ch.dm

```diff
@@ -0,0 +1,864 @@
+		"You frantically writhe upwards a couple of inches, before the beast swallows with a sloppy-sounding glllggk, sending you back down a foot or so. Each struggle you make only seems to hasten your journey down the hatch. ")
+	autotransferlocation = "Stomach"
+
+/obj/belly/dragon/throat/heal
+	name = "Throat."
+	desc = "Say while you're here and sliding down a dragons throat, I figure you must've figured out how to tame this dragon by reading the PR, seeing as these flavourtexts will be gone in less than 12h."
+	escapechance = 100
+	digest_mode = DM_HEAL
+	struggle_messages_inside = list(
+		"You outstretch your limbs back up the throat from whence you came, trying to pry yourself back up. A difficult challenge without aid. The dragon, sensing your desire to come back out, is quick to aid. Hurking you back up its gullet, promptly depositing you neatly on the ground below itself.")
+	autotransferlocation = "Stomach."
+
+/obj/belly/dragon/stomach
+	name = "Stomach"
+	desc = "The final part of your journey arrives, in the form of a tightly squelched, muscular sphincter. Throat pressure crams against you, until abruptly, you find yourself spat into a hot, caustic cauldron of churning doom, the dragon's stomach. After slithering in, the way you entered squelches shut, dissapearing among the folds - impossible for you to find any more. You are trapped, no way out, lounging in a shallow pool of thick sticky juices. endless undulations from thick, pendulous folds of stomach-wall all around continually churn it up into a foamy, bubbling mess, soaking their folds in it to drip in ropes and even shivering sheets of the stuff around you. Clenches gush the digestive slimes all over you from time to time, cradling you in it's noxious embrace. Your ears are filled with such sloppy squelches now, those distant muffled glrrns you heard earlier now sharp, crisp, and thunderous as you nestle in their very source. Settle down for what little time you have left, for your fate rests adding to the powerful beast all around you."
+	vore_sound = "Tauric Swallow"
+	escapable = 0
+	//desc = "The dragon's cavernous stomach hungrily pries your figure in through its sphincter, sealing taught behind you. Its meal is hastily subject to the slick, squelching confines of its digestive organ, that hanging gut eagre to churn and ripple over your proportions the moment you've arrived. Caustic waves of slop and chyme wash over you, that acidic tingle setting in upon your flesh. The entire chamber groans and echoes with each of the dragon's prideful steps. Satiated with its most recent meal, slowly being reduced into a nutritious paste to further power its hunt. Your fate definitively sealed within this tomb of swelteringly hot flesh."
```

Comment:
_squint_

Reply:
Still fluffing out the vore organ descriptions and such.